client/reader/search-stream/index.jsx

```diff
@@ -229,6 +236,9 @@ class SearchStream extends React.Component {
 								/>
 							</div>
 						) }
+						{ query && ! (showPostsColumn || showSitesColumn) &&
```

Comment:
There are a few minor formatting issues that Prettier should fix for you (like the spacing inside the brackets here). Have you tried setting up wp-prettier (our Prettier fork) in your editor?

https://github.com/Automattic/wp-prettier

There's also a pre-commit hook that should do this formatting for you.

Reply:
@bluefuton I'll get this fixed.

Reply:
This is fixed @bluefuton